src/lib/mixin.ts

```diff
@@ -59,11 +59,15 @@ export function MobxReactionUpdate<T extends UpdatingElementConstructor>(
             }
         }
 
-        protected performUpdate(): void {
+        protected update(
+            _changedProperties: Map<string | number | symbol, unknown>
```

Comment:
```suggestion
            _changedProperties: PropertyValues
```
Do we prefer the type from `LitElement`?

Reply:
Yes! fixed in fb55b9d